packages/create-pantheon-decoupled-kit/src/templates/next-drupal-search-api-addon/app/search-input.jsx

```diff
@@ -0,0 +1,39 @@
+import { useRouter } from 'next/navigation';
+import { useState } from 'react';
+
+const SearchInput = () => {
+	const [searchQuery, setSearchQuery] = useState('');
+
+	const router = useRouter();
+
+	const onSearch = (event) => {
+		event.preventDefault();
+
+		if (typeof searchQuery !== 'string') {
+			return;
+		}
+
+		const encodedSearchQuery = encodeURI(searchQuery);	
+		router.push(`/search?=${encodedSearchQuery}`);
+	};
+
+	return (
+		<form onSubmit={onSearch}>
+			<input
```

Comment:
Form inputs should have a `label` element for a11y reasons. I am not 100%, but we might also want some aria labels and attributes on this markup